docs/reference/cat/allocation.asciidoc

```diff
@@ -2,7 +2,7 @@
 == cat allocation
 
 `allocation` provides a snapshot of how shards have located around the
-cluster and the state of disk usage.
+cluster and the state of disk usage. This call lists only data nodes in cluster by default.
```

Comment:
This implies that you can change the default.  How about:

    `allocation` provides a snapshot of how many shards are allocated to each data node
    and how much disk space they are using.